Find a functionally equivalent code:android.content.AsyncTaskLoader.LoadTask.onPostExecute:COMMENT
Method Modifier: protected   
Comment:/* Runs on the UI thread */

@@ -1,6 +1,6 @@
 {
     if (DEBUG)
-        Slog.v(TAG, this + " onPostExecute");
+        Log.v(TAG, this + " onPostExecute");
     try {
         AsyncTaskLoader.this.dispatchOnLoadComplete(this, data);
     } finally {

